Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -57,6 +64,23 @@ public class MXRAndroidSystem : IMXRSystem {
             int WIFI_CONNECTION_STATUS = 3000;
             int RUNTIME_SETTINGS = 4000;
             int DEVICE_STATUS = 5000;
+            int HANDLE_COMMAND = 6000;
+            int GET_HOME_SCREEN_STATE = 15000;
+
+            // Subscribe to application focus change event and 
+            // execute any command passed in extra strings
+            Dispatcher.OnPlayerFocusChange += x => {
+                if(x) {
+                    var intent = MXRAndroidUtils.CurrentActivity.Call<AndroidJavaObject>("getIntent");
+                    var commandJson = intent.Call<string>("getStringExtra", "command");
+
+                    // TODO: Will this cause the same command to be executed
```

Comment:
Good question! 

"Will the admin app clear extra strings by itself" - I don't think so. IIUC, once we launch the activity with an intent, it has those arguments. So if we've already run this command, we don't want to run it again - otherwise every time we have a focus change, we'll rerun the same command, right?

If we need to add some kind of "command hash" to this based on a timestamp, we can - let me know what you think!

Reply:
Yeah I think a timestamp value or a hash based on it would be great. Should we just add it as an extra string? So `adb ... -es command <json> commandHash <hash>?`

Reply:
You can use the function "MXRHomeScreenAndroidUtils.GetIntentId()" to get the intentId, which we will pass to you from the Android side as part of this design and then cache it, to confirm we are not rerunning the same command.

Reply:
Updates in 8ac556c0642e8ce425c6be96f491f5231010b5a5. The SDK now remembers intentId of intent commands in a hashset and ignores those ids after that.